ru_RU.lang

```diff
@@ -66,7 +66,7 @@ replaymod.gui.videothumbnailtoolarge=Размер изображения пре
 replaymod.gui.videothumbnailformat=Изображение должно быть в формате JPEG или PNG
 
 replaymod.gui.hours=ч
-replaymod.gui.minutes=мин
+replaymod.gui.minutes=мин.
```

Comment:
Is the dot really required/preferred when abbreviating "minutes"? Seems odd that it's only on minutes but not on hours, seconds and milliseconds.

Reply:
Left by an accident